js/apps/admin-ui/test/utils/file-chooser.ts

```diff
@@ -1,11 +1,15 @@
 import { Page } from "@playwright/test";
 import path from "path";
+import { fileURLToPath } from "url";
```

Comment:
Nit: We should use the `node:` prefix for imports that come from Node's standard library. e.g. also `path` should be `node:path`.
```suggestion
import { fileURLToPath } from "node:url";
```